libs/tendermint/README.md

```diff
@@ -0,0 +1,7 @@
+# tendermint
+
+This library was generated with [Nx](https://nx.dev).
```

Comment:
**nitpick (non-blocking)**: Lazy default readme 😛 

Nice move though. There'll be some block explorer stuff that can move out to here for sure.